### PR TITLE
Update Charpoly

### PR DESCRIPTION
**Pull request by latot**
Fixes https://github.com/cbm755/octsympy/issues/531

Left update the doc with examples, but first this https://github.com/cbm755/octsympy/pull/538 or we will have branch problems.

### CODE REVIEW

File: inst/@sym/charpoly.m [hunk 1]
```diff
@@ -54,11 +54,17 @@
   if (nargin >= 3)
     print_usage ();
   end
+
+  cmd = {'return _ins[0].charpoly(_ins[1]).as_expr(),'};
+
   if (nargin == 1)
-    error('Charpoly as vector its not supported now');
+    syms b
+    y = python_cmd(cmd , sym(a), b);
```
Comment (cbm755) [inline inst/@sym/charpoly.m]: does this cause some problem if `b` is in the matrix already?  Also, given #543, maybe we should just do some python code here.  Then we can use `b = Dummy()` for the temp variable.

Or better yet, there must be a command other than `as_expr` which extracts the coefficients into a list...

**Comment by cbm755** [comment]
Also, will need another example.

File: inst/@sym/charpoly.m [hunk 2]
```diff
@@ -65,23 +65,42 @@
 %%       λ  - 2⋅λ⋅x + x  - x
 %% @end group
 %% @end example
+%%
+%% If you don't send any @var{x}, you will get a vector with the
+%% coefficients of the polynomi:
```
Comment (cbm755) [inline inst/@sym/charpoly.m]: Change "send" to "pass in".  Polynomial is spelled wrong.

Comment (latot) [inline inst/@sym/charpoly.m]: Okidokis, Done

File: inst/@sym/charpoly.m [hunk 3]
```diff
@@ -65,23 +65,42 @@
 %%       λ  - 2⋅λ⋅x + x  - x
 %% @end group
 %% @end example
+%%
+%% If you don't pass in any @var{x}, you will get a vector with the
+%% coefficients of the polynomi:
```
Comment (cbm755) [inline inst/@sym/charpoly.m]: polynomial still misspelled, I'll fix it after merge

**Pull request merged**